website/content/docs/configuration/events/telemetry-data.mdx

```diff
@@ -0,0 +1,586 @@
+---
+layout: docs
+page_title: Telemetry data
+description: >-
+  Learn what data Boundary captures when telemetry is enabled and how you can view it.
+---
+
+# Telemetry data
+
+If you have telemetry enabled in Boundary, you can view information about the following resources.
+
+## Auth methods
+
+<table>
+    <thead>
+        <tr>
+            <th>Events</th>
+            <th>Telemetry fields</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td rowspan="7">Auth methods (Create)</td>
+            <td>Auth method ID</td>
+        </tr>
+        <tr>
+            <td>Auth method type</td>
+        </tr>
+            <tr>
+            <td>Auth method URI</td>
+        </tr>
+        <tr>
+            <td>Link (Includes location: org ID, project ID, and internal ID)</td>
```

Comment:
I believe the link is only available to us because we append them into the telemetry events when we are processing them in cloud boundary service.  so boundary user wont see this information when they turn on the telemetry